### PR TITLE
cc: CUBIC - redefine K equation

### PR DESCRIPTION
**Pull request by junhochoi**
- Update based on https://github.com/NTAP/rfc8312bis/pull/3
- Fix congestion avoidance test

### CODE REVIEW

File: src/recovery/cubic.rs [hunk 1]
```diff
@@ -463,32 +476,32 @@ mod tests {
...
-        }
-
         // Trigger congestion event to update ssthresh
         r.congestion_event(now, packet::EPOCH_APPLICATION, now);
 
         // After congestion event, cwnd will be reduced.
         let cur_cwnd = (prev_cwnd as f64 * BETA_CUBIC) as usize;
         assert_eq!(r.cwnd(), cur_cwnd);
 
+        // Shift current time to 1 RTT
```
Comment (LPardue) [inline src/recovery/cubic.rs:486]:
```suggestion
        // Shift current time by 1 RTT.
```

Just an english language thing, unless you really mean that you want to shift to 100ms after the big bang :D

File: src/recovery/cubic.rs [hunk 2]
```diff
@@ -463,32 +476,32 @@ mod tests {
...
-            pkt_num: 0,
-            // To exit from recovery
-            time_sent: now + rtt,
-            size: r.max_datagram_size,
-        }];
+        // During Congestion Avoidance, it will take
+        // 3 ACKs (40x bytes per ACK) to increase cwnd by 1 MSS.
+        for _ in 0..3 {
+            let now = now + rtt;
+            let acked = vec![Acked {
```
Comment (LPardue) [inline src/recovery/cubic.rs:495]: More of a general observation but it seems like we could save some lines (and potential copy/paste errors) across the codebase if we had an `Acked::new()` method.

Comment (junhochoi) [reply src/recovery/cubic.rs:495]: or `Copy` trait?

File: src/recovery/cubic.rs [hunk 3]
```diff
@@ -26,8 +26,8 @@
 
 //! CUBIC Congestion Control
 //!
-//! This implementation is based on the following RFC:
-//! <https://tools.ietf.org/html/rfc8312>
+//! This implementation is based on the following draft:
+//! <https://tools.ietf.org/html/draft-eggert-tcpm-rfc8312bis-02>
```
Comment (ghedo) [inline src/recovery/cubic.rs:30]:
```suggestion
//! <https://tools.ietf.org/html/draft-ietf-tcpm-rfc8312bis-02>
```

**Review by ghedo — DISMISSED** [review]
Added minor nit, but lgtm.

**Pull request merged**